client/src/components/BlockLinkField/tests/BlockLinkField-test.js

```diff
@@ -1,305 +1,111 @@
-        <BlockLinkField
-          {...props}
-        />
-      );
-      const event = {
-        target: {
-          type: 'button'
+test('BlockLinkField should return the linked page within the Entwine context', () => {
+  const { container } = render(
+    <BlockLinkField {...makeProps({
+      linkedPage: {},
+      data: {
+        linkedPage: {
+          URLSegment: '/react',
         }
-      };
-
-      wrapper.setState({ modalOpen: false });
-      wrapper.instance().handleClick(event);
-
-      expect(wrapper.state('modalOpen')).toEqual(false);
-    });
-  });
-
-  describe('registerChange()', () => {
-    it('', () => {
-      const wrapper = shallow(
-        <BlockLinkField
-          {...props}
-        />
-      );
-
-      wrapper.setState({ isDirty: false });
-      wrapper.instance().registerChange();
-
-      expect(wrapper.state('isDirty')).toEqual(true);
-    });
-  });
-
-  describe('getLinkedPage()', () => {
-    it('should return the linked page within the Entwine context', () => {
-      const wrapper = shallow(
-        <BlockLinkField
-          {...props}
-        />
-      );
-      wrapper.setProps({
-          linkedPage: {},
-          data: { linkedPage: '/react' }
-        });
-
-      expect(wrapper.instance().getLinkedPage()).toEqual('/react');
-    });
-
-
-    it('should return the linked page within the React context', () => {
-      const wrapper = shallow(
-        <BlockLinkField
-          {...props}
-        />
-      );
-      wrapper.setProps({ linkedPage: { URLSegment: '/entwine' } });
-
-      expect(wrapper.instance().getLinkedPage()).toEqual({ URLSegment: '/entwine' });
-    });
-  });
-
-  describe('getLinkRelativeUrl()', () => {
-    it('returns relative URL for the linked page with a leading slash', () => {
-      const wrapper = shallow(
-        <BlockLinkField
-          {...props}
-        />
-      );
-
-      wrapper.instance().getLinkedPage = mockGetLinkedPage.mockReturnValue('/mockUrlWithSlash');
-      expect(wrapper.instance().getLinkRelativeUrl()).toEqual('/mockUrlWithSlash');
-
-      wrapper.instance().getLinkedPage = mockGetLinkedPage.mockReturnValue('/mockUrlWithOutSlash');
-      expect(wrapper.instance().getLinkRelativeUrl()).toEqual('/mockUrlWithOutSlash');
-    });
-
-    it('returns empty string if no relative URL provided', () => {
-      const wrapper = shallow(
-        <BlockLinkField
-          {...props}
-        />
-      );
+      },
+    })}
+    />
+  );
+  expect(container.querySelector('.block-link-field__link').textContent).toBe('/react')
+});
 
-      wrapper.instance().getLinkedPage = mockGetLinkedPage.mockReturnValue(null);
-      expect(wrapper.instance().getLinkRelativeUrl()).toEqual('');
-    });
-  });
+test('BlockLinkField should return the linked page within the React context', () => {
+  const { container } = render(
+    <BlockLinkField {...makeProps({
+      linkedPage: {
+        URLSegment: '/entwine',
+      },
+      value: '{"PageID":1}'
```

Comment:
```suggestion
```
Already the default value. Also doesn't seem relevant - not included in the original test.